### ISSUE TITLE
[BUG] MediaElement's PositonChanged event not firing on Android

### ISSUE THREAD

**Issue by yusuftuncay** [CommunityToolkit/Maui#1606]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [X] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

MediaElement's "PositionChanged" event doesn't fire on Android, but it does work on Windows

### Expected Behavior

MediaElement's "PositionChanged" event has to fire when media is playing and the position is changed

### Steps To Reproduce

1. Add a MediaElement and attach a PositionChanged event, let it generate a method: 
```xaml
<toolkit:MediaElement
    Source="https://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4"
    PositionChanged="MediaElement_PositionChanged">
    
</toolkit:MediaElement>
```

```csharp
private void MediaElement_PositionChanged(object sender, MediaPositionChangedEventArgs e)
{
    var x = e.Position;
}
```

2. Set a breakpoint in the "PositionChanged" method
3. It **won't** hit on Android, it **will** hit on Windows

### Link to public reproduction project repository

https://github.com/yusuftuncay/MediaElementPositionTest

### Environment

```markdown
- OS: Android
- .NET 8
- Visual Studio 17.8

<PackageReference Include="CommunityToolkit.Maui" Version="7.0.1" />
<PackageReference Include="CommunityToolkit.Maui.MediaElement" Version="3.0.0" />
```

### Anything else?

I have not found a fix yet, nothing online or me trying out anything possible.

**Comment by bijington** [comment]
I believe this was fixed in #1555 and it should be in release `3.0.1`

**Comment by yusuftuncay** [comment]
Thanks a lot!
